core/src/main/java/hudson/triggers/SCMTrigger.java

```diff
@@ -119,6 +132,23 @@ public boolean isIgnorePostCommitHooks() {
         return this.ignorePostCommitHooks;
     }
 
+    /**
+     * Data-bound setter for ignoring post commit hooks.
+     *
+     * @param ignorePostCommitHooks
+     *     True if we should ignore post commit hooks, false otherwise.
+     *
+     * @since 2.21
+     */
+    @DataBoundSetter
+    public void setIgnorePostCommitHooks(boolean ignorePostCommitHooks) {
+        this.ignorePostCommitHooks = ignorePostCommitHooks;
+    }
+
+    public String getScmpoll_spec() {
```

Comment:
Well it is just the required matching getter. But :ant: as part of adding this, you should delete the `value` [here](https://github.com/jenkinsci/jenkins/blob/f9c78d07da136f424650b01ced270e77b479e05c/core/src/main/resources/hudson/triggers/SCMTrigger/config.jelly#L28).

Pity that we cannot fix the awful field name while we are at it, but that would be incompatible for anyone using

```groovy
[$class: 'SCMTrigger', scmpoll_spec: '@daily']
```

syntax. What I think we need (and I have a longer discussion of this somewhere on jenkinsci-dev) is an API in `symbol-annotation` allowing a class to completely override its `DescribableModel` introspection with custom logic, à la subclassing `BeanInfo`. This would allow us to define a more sensible name for the field, such as simply `spec`, while still retaining compatibility for calls to `instantiate` using `scmpoll_spec`.